feconf.py

```diff
@@ -641,7 +641,10 @@ def get_empty_ratings():
 
 # Bool to enable update of analytics models.
 ENABLE_NEW_STATS_FRAMEWORK = False
-# Current event models schema version.
+# Current event models schema version. All event models with
+# event_schema_version as 1 are the events collected before the rework of the
+# statistics framework which brought about the recording of new event models.
+# This tentatively includes all models recorded before Nov 2017.
```

Comment:
Drop "tentatively"; just fix the date later if needed. (Unless "tentatively" refers to something else?)

Reply:
It was referring to somewhere around Nov 2017 - but you're right. Lets fix the date later.